batch7crud-roll9/src/main/java/com/lftechnology/batch7crud/util/DBConnection.java

```diff
@@ -0,0 +1,48 @@
+package com.lftechnology.batch7crud.util;
+
+import com.lftechnology.batch7crud.exception.DataException;
+
+import javax.naming.Context;
+import javax.naming.InitialContext;
+import javax.naming.NamingException;
+import javax.sql.DataSource;
+import java.io.Closeable;
+import java.io.IOException;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**
+ * Created by sanjay on 1/18/16.
+ */
+public class DBConnection implements AutoCloseable{
+  private static Connection conn = null;
+  private static final Logger LOGGER = Logger.getLogger("DBConnectionLog");
+
+  private DBConnection() {
+  }
+
+  public static Connection getConnection() throws DataException {
+    try {
+      Context initContext = new InitialContext();
+      Context envContext = (Context) initContext.lookup("java:/comp/env");
+      DataSource ds = (DataSource) envContext.lookup("jdbc/db_userinfo_test");
+      conn = ds.getConnection();
+      return conn;
+    } catch (NamingException ex) {
+      LOGGER.log(Level.SEVERE, ex.getMessage(), ex);
+      throw new DataException();
+    } catch (SQLException ex) {
+      LOGGER.log(Level.SEVERE, ex.getMessage(), ex);
+      throw new DataException();
+    }
+  }
+
+  @Override
+  public void close() throws Exception {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1186)